x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/bucket_nesting_editor.tsx

```diff
@@ -127,7 +127,7 @@ export function BucketNestingEditor({
                 defaultMessage: 'Entire data set',
               }),
             },
-            ...aggColumns,
+            ...aggColumns.map(({ value, text }) => ({ value, text })),
```

Comment:
This actually fixes a console error that @cchaos found, it's not directly related

Reply:
💯